Kernel_23/test/Kernel_23/include/CGAL/_test_new_3.h

```diff
@@ -747,6 +747,7 @@ test_new_3(const R& rep)
         = rep.oriented_side_3_object();
   Oriented_side tmp39 = oriented_side(h2,p2);
                 tmp39 = oriented_side(sp9,p2);
+                tmp39 = oriented_side(p2,v1,p2);
```

Comment:
This pull-request, and particularly this line, is responsible for the error seen in the testsuite here:
https://cgal.geometryfactory.com/CGAL/testsuite/results-5.3-Ic-91.shtml#Kernel_23

The reason is that the `Vector_3` `v1` is never initialized, so it can be anything from what is in memory. If one of its coordinates is a `NaN`, that triggers a SIGFPE once that `NaN` is converted to a GMP object.

Here is the backtrace I got:
```
#0  0x00007ffff7795ef5 in raise () from /usr/lib/libc.so.6
#1  0x00007ffff7c7539f in __gmp_invalid_operation () from /usr/lib/libgmp.so.10
#2  0x00007ffff7c912b6 in __gmpq_set_d () from /usr/lib/libgmp.so.10
#3  0x00005555555a4c18 in __gmp_expr<__mpq_struct [1], __mpq_struct [1]>::assign_d (d=-nan(0xfffffffffffff), this=0x7fffffffdb20) at /usr/include/gmpxx.h:1777
#4  __gmp_expr<__mpq_struct [1], __mpq_struct [1]>::init_d (d=-nan(0xfffffffffffff), this=0x7fffffffdb20) at /usr/include/gmpxx.h:1781
#5  __gmp_expr<__mpq_struct [1], __mpq_struct [1]>::__gmp_expr (d=-nan(0xfffffffffffff), this=0x7fffffffdb20) at /usr/include/gmpxx.h:1809
#6  CGAL::NT_converter<double, __gmp_expr<__mpq_struct [1], __mpq_struct [1]> >::operator() (a=<optimized out>, this=<optimized out>) at /cgal/Number_types/include/CGAL/NT_converter.h:33
#7  CGAL::Cartesian_converter<CGAL::Type_equality_wrapper<CGAL::Cartesian_base_no_ref_count<double, CGAL::Epick>, CGAL::Epick>, CGAL::Simple_cartesian<__gmp_expr<__mpq_struct [1], __mpq_struct [1]> >, CGAL::NT_converter<double, __gmp_expr<__mpq_struct [1], __mpq_struct [1]> > >::operator() (a=..., this=<optimized out>) at /cgal/Cartesian_kernel/include/CGAL/Cartesian_converter.h:300
#8  0x00005555555fa662 in CGAL::Filtered_predicate<CGAL::CommonKernelFunctors::Oriented_side_3<CGAL::Simple_cartesian<__gmp_expr<__mpq_struct [1], __mpq_struct [1]> > >, CGAL::CommonKernelFunctors::Oriented_side_3<CGAL::Simple_cartesian<CGAL::Interval_nt<false> > >, CGAL::Cartesian_converter<CGAL::Type_equality_wrapper<CGAL::Cartesian_base_no_ref_count<double, CGAL::Epick>, CGAL::Epick>, CGAL::Simple_cartesian<__gmp_expr<__mpq_struct [1], __mpq_struct [1]> >, CGAL::NT_converter<double, __gmp_expr<__mpq_struct [1], __mpq_struct [1]> > >, CGAL::Cartesian_converter<CGAL::Type_equality_wrapper<CGAL::Cartesian_base_no_ref_count<double, CGAL::Epick>, CGAL::Epick>, CGAL::Simple_cartesian<CGAL::Interval_nt<false> >, CGAL::NT_converter<double, CGAL::Interval_nt<false> > >, true>::operator()<CGAL::Point_3<CGAL::Epick>, CGAL::Vector_3<CGAL::Epick>, CGAL::Point_3<CGAL::Epick> > (this=<optimized out>) at /cgal/Number_types/include/CGAL/FPU.h:558
#9  0x000055555589d298 in test_new_3<CGAL::Epick> (rep=...) at /cgal/Kernel_23/test/Kernel_23/include/CGAL/_test_new_3.h:750
#10 0x000055555589f149 in test<CGAL::Epick> () at /cgal/Kernel_23/test/Kernel_23/Filtered_cartesian.cpp:96
#11 0x000055555559d5f3 in main () at /cgal/Kernel_23/test/Kernel_23/Filtered_cartesian.cpp:75
```

and at the frame #9, here is the variable `v1`:
```
#9  0x000055555589d298 in test_new_3<CGAL::Epick> (rep=...) at /cgal/Kernel_23/test/Kernel_23/include/CGAL/_test_new_3.h:750
750                     tmp39 = oriented_side(p2,v1,p2);
(gdb) p v1
$11 = {<CGAL::VectorC3<CGAL::Epick>> = {base = {_M_elems = {-nan(0xfffffffffffff), 7.5189695826554899e-304, 6.9533558074674195e-310}}}, <No data fields>}
```
It has a `NaN` (`0xfffffffffffff`) as its first coordinate.